### ISSUE TITLE
Kissanime does not work for the last three days

### ISSUE THREAD

**Issue by endlessterror** [HIGHWAY99/plugin.video.kissanime#13]
Tried on 3 different Kodi 16.1 platforms. Whatever is selected, an error is thrown back, something about "Movies blah blah"

Addon was awesome, I hope you can fix it :)

**Comment by Blood420** [comment]
From what I understand it needs an update since they changed both servers and the dot com Is now dot io  but highway99 is taking care of family matters b/c of his mothers death with that said sry for ur loss highway99 but for the time just anime stream is an OK replacement but the only way u can find anime on it is by searching for it by name its not the greatest but at least it Simi works

**Comment by endlessterror** [comment]
Yeah, after opening this thread I checked the IRC channel and found out about this event. I've offered my condolences in the reddit forum, I also offer them here...

As for the change, I had the time to examine the situation and indeed it seems to be the DNS change. I've remotely changed all referecences of kissanime.to to kissanime.io in the 3 files referenced (default.py, config.py and teh_tools.py). Unfortunately, not at home atm, so I can not test :)

If you are interested I can upload these files and you can test. Or wait for me to test and get back with the results.

**Comment by Blood420** [comment]
yes that would be great and show me how to change it or explain it either way because i run my kodi off the fire stick and the fire tv and thank you

**Comment by endlessterror** [comment]
Unfortunately I do not have good news. I modified the 3 files I mentioned above, however the problem persists. It seems that there are other changes as well.

For the record what I did is search and replace all references of "kissanime.to" to "kissanime.io", in files default.py, config.py and teh_tools.py.

These 3 files are in the addons/plugin.video.kissanime directory. I am sorry, but I do not know how you can access these files on the firestick.

The code perhaps could be modified, but being a complete newbie in Python, as well as in parsing html pages from sites certainly does not help me provide a complete fix here :)

**Comment by Blood420** [comment]
no worries thanks for the help any way i guess we'll have to wait on highway it is what it is

**Comment by endlessterror** [comment]
Well, after searching around it seems that the .io domain is a scam :/ Avoid it like hell, see also https://www.reddit.com/r/KissAnime/comments/5hlzpw/faq/

Plus, there are some comments on the official Kissanime Facebook fanpage stating server issues: https://www.facebook.com/kissanimefanpage/

**Comment by Blood420** [comment]
Well that sux but thanks

**Comment by aanderson144** [comment]
I have fixed it just replace every instance of"kissanime.to" to "kissanime.ru" in these files config.py, default.py and teh_tools.py and then delete your user cache

**Comment by aanderson144** [comment]
I have a zip install but no place to put it it is pretty easy to do though

**Comment by aanderson144** [comment]
should have read more someone already did it yay drunk not reading

**Comment by aanderson144** [comment]
wait he did the .io well the real new site is .ru remember that

**Comment by aanderson144** [comment]
either way anime is the best!!!!!!!!!!!!!!

**Comment by Blood420** [comment]
Let me know where I can get ur fix plz

**Comment by aanderson144** [comment]
heres a link to the zip remember you need to delete your user cache http://www.filehosting.org/file/details/628091/plugin.video.kissanime.zip

**Comment by endlessterror** [comment]
@Cidd144 : do note that IMHO, after searching all occurences of kissanime.to and replacing them kissanime.ru, you should also search for kissanime.com and replace them with kissanime.to

This is needed (AFAIK) in order to make sure that existing bookmarks/entries pointing to *.to will be converted to *.ru

My 5 cents :)
